packages/hs-react-ui/src/components/Button/Button.stories.tsx

```diff
@@ -39,7 +39,7 @@ storiesOf('Button', module)
         <Button
           variant={select('variant', variants, variants.fill)}
           color={color('color', colors.primaryDark)}
-          onClick={action('button-click')}
+          onClick={() => action('button-click')()}
```

Comment:
is this necessary? action() returns a function which should get called by onClick - here you're passing a function which fires the function returned from action(). It's a weird case but I feel like the old version should have been correct

Reply:
Yep, you are right. Switching it back.